src/main/java/rx/internal/util/LambdaNotificationObserver.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright 2016 Netflix, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package rx.internal.util;
+
+import rx.*;
+import rx.functions.*;
+
+/**
+ * An Observer that forwards the onXXX method calls to a notification callback
+ * by transforming each signal type into Notifications.
+ * @param <T> the value type
+ */
+public final class LambdaNotificationObserver<T> implements Observer<T> {
```

Comment:
I would think these "Lambda"-prefix classes should be "Action"-prefixed since lambdas are language implementation details for creating Action instances but otherwise have nothing to do with the implementation of these classes.